### ISSUE TITLE
[Help wanted] Update Emoji

### ISSUE THREAD

**Issue by Fintasys** [Fintasys/emoji_picker_flutter#1]
I'm looking for help to update the emojis. If you have any good idea let me know or want to contribute feel free to do it !

**Comment by poitch** [comment]
The react equivalent of emoji_picker is using https://github.com/iamcal/emoji-data

**Comment by Fintasys** [comment]
@poitch Great hint, thank you ! Gonna check it out 🙇

**Comment by LostInDarkMath** [comment]
What exactly is to be done here?

**Comment by Fintasys** [comment]
@LostInDarkMath Well, best case would be to write some automation to update the list the of emoji's based on some more up-to-date source. But not just adding emoji's is important but also to add them at the right place. So far I was manually adjusting the positions & order based on WhatsApp's Emoji Picker.

**Comment by Fintasys** [comment]
@LostInDarkMath Yes, supported emoji's increase over time and with each os update. Also the [Unicode list](https://unicode.org/emoji/charts/full-emoji-list.html) of emoji's is being updated regularly (currently version 14)
